docs/BackupAndRestore.md

```diff
@@ -0,0 +1,594 @@
+# NuoDB Backup and Restore in Kubernetes
+
+## Introduction
+
+NuoDB provides several automated mechanisms for backing up and restoring a database in a Kubernetes environment when deploying NuoDB using the NuoDB Helm Charts.
+
+After database installation, the available backup and restore features supported by these mechanisms are: 
+
+1. [Automatic archive initial import](#automatic-archive-initial-import)
```

Comment:
is the initial import done "after database installation"?

Reply:
It is done after `database` chart installation by `nuosm` before SMs database processes are started.